src/test/resources/features/Sessions.feature

```diff
@@ -0,0 +1,315 @@
+    And click on continue button
+    Then Im on "Manage prison visits - Who is the main contact for this booking?" page
+    And I select the main contact option
+    And I enter "<phoneNumber>" to get text message
+    And click on continue button
+    Then Im on "Manage prison visits - How was this booking requested?" page
+    And I select a phone call option on method used to make the reqest
+    And click on continue button
+    Then Im on "Manage prison visits - Check the visit details before booking" page
+    And click on continue button
+    #And click on submit booking button
+    Then Im on "Manage prison visits - Booking confirmed" page
+    And I see "Booking confirmed" message displayed
+    And I see a booking reference
+    And I click on Manage a Prison option
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I click on Book a visit option
+    And Im on "Manage prison visits - Search for a prisoner" page
+    When I enter "<prisonNumber>" to search for a prison
+    And click on search button
+    Then I choose prison from search results
+    Then Im on "Manage prison visits - Vsip_prisoner12, Do Not Use" page
+    And I select last booked visit reference
+    And I check "open" session is booked
+    And I sign out of the service
+    #Reverting the Booking for consistent Test execution
+    And I want to clean up after the above test
+
+    Examples:
+      | userName  | password          | prisonNumber | phoneNumber    |
+      | VSIP4_TST | DigitalServices10 | A8897DZ      | 07806543211    |
+
+  @suite
+  Scenario Outline: Closed session - User able to book a Closed session for all Prisoners within the establishment
+    Given I navigate to VSIP service
+    And Im on "HMPPS Digital Services - Sign in" page
+    And I enter "<userName>"
+    And I enter the "<password>"
+    And click on submit button
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I select change establishment
+    And I change the establishmnet to Drake Hall
+    And click on continue button
+    And I click on Book a visit option
+    And Im on "Manage prison visits - Search for a prisoner" page
+    When I enter "<prisonNumber>" to search for a prison
+    And click on search button
+    Then I choose prison from search results
+    Then Im on "Manage prison visits - Vsip_prisoner11, Do Not Use" page
+    And I click on Book a visit button
+    Then Im on "Manage prison visits - Select visitors from the prisoner’s approved visitor list" page
+    And I select a visitor form the list from Drake hall establishment
+    And click on continue button
+    Then Im on "Manage prison visits - Check the prisoner&#39;s closed visit restrictions" page
+    And I choose closed type visit
+    And click on continue button
+    Then Im on "Manage prison visits - Select date and time of visit" page
+    #Closed type is displayed
+    And I check "Closed" visit type is displayed
+    And I select time slot
```

Comment:
Same as above. Selecting the first available time slot does not ensure that no closed sessions are being shown for that prisoner? The aim of the test is to ensure that no closed sessions are available to that prisoner.

Reply:
same as above